Solutions/NXLogAixAudit/Parsers/NXLog_parsed_AIX_Audit_view.txt

```diff
@@ -0,0 +1,33 @@
+// Usage Instructions:
+// Paste the query below into the Log Analytics query editor.
+// Click the "Save" button and select "Save as function".
+// Enter "NXLog_parsed_AIX_Audit_view" in the "Function name" field.
+// For "Legacy category:" enter "AIX Security Audit".
+// "Paramters" are not needed.
+// Function usually takes 10-15 minutes to activate.
+// You can then use this function from any other queries (e.g. NXLog_parsed_AIX_Audit_view | take 10).
+// Reference: Using functions in Azure monitor log queries : https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+let NXLog_parsed_AIX_Audit_view = view () {
+  AIX_Audit_CL
+  | project-rename
+    Command=Command_s,
+    EventReceivedTime=EventReceivedTime_t,
+    EventTime=EventTime_t,
+    EventType=EventType_s,
+    Hostname=Hostname_s,
+    Login=Login_s,
+    LoginUID=LoginUID_d,
+    MessageSourceAddress=MessageSourceAddress_s,
+    ParentPID=ParentPID_d,
+    PID=PID_d,
+    Real=Real_s,
+    RealUID=RealUID_d,
+    SourceModuleName=SourceModuleName_s,
+    SourceModuleType=SourceModuleType_s,
+    Status=Status_d,
```

Comment:
"EventResultDetails"

Reply:
Resolved with changes in [0a0e387](https://github.com/Azure/Azure-Sentinel/pull/2968/commits/0a0e3879f0d261b33e4dd72283fe919f75852ffe)